incubator/seq/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+name: seq
+version: 0.1.0
+appVersion: 5.0.832-pre
+description: Seq is the easiest way for development teams to capture, search and visualize structured log events! This page will walk you through the very quick setup process.
```

Comment:
The last sentence of the description probably doesn't make sense here.

Reply:
Fixed